examples/advanced_operations/add_responsive_search_ad_full.py

```diff
@@ -484,23 +583,16 @@ def add_images(client, customer_id, campaign_resource_name):
 
     # Step 6.2 - Create Image Extension
     extension_feed_item_service = client.get_service("ExtensionFeedItemService")
-    extension_feed_item_operation = client.get_type(
-        "ExtensionFeedItemOperation"
-    )
+    extension_feed_item_operation = client.get_type("ExtensionFeedItemOperation")
     extension_feed_item = extension_feed_item_operation.create
     extension_feed_item.image_feed_item.image_asset = image_asset_resource_name
 
-    extension_feed_response = (
-        extension_feed_item_service.mutate_extension_feed_items(
-            customer_id=customer_id, operations=[extension_feed_item_operation]
-        )
+    extension_feed_response = extension_feed_item_service.mutate_extension_feed_items(
+        customer_id=customer_id, operations=[extension_feed_item_operation]
     )
     image_resource_name = extension_feed_response.results[0].resource_name
 
-    print(
-        "Created an image extension with resource name: "
-        f"'{image_resource_name}'"
-    )
+    print("Created an image extension with resource name: " f"'{image_resource_name}'")
 
     # Step 6.3 - Link Image Extension to RSA
```

Comment:
Seems it's actually adding it to the Campaign, is that right? 
And is this still the right way to do it with the ImageAsset migration being complete?

Reply:
@bobhancockg Can you say what the correct workflow is here?

Reply:
According to [these docs](https://developers.google.com/google-ads/api/docs/assets/working-with-assets#use_the_asset_in_an_ad) RSAs only support text assets, so this whole aspect of the example might not be feasible.

Reply:
So the portion of this example that creates image assets should be removed and replaced with text assets that are created inline along with the Ad. @bobhancockg if that sounds good I'll make these changes for a future release.

Reply:
Looks like we already set text assets on the AdGroupAd in `create_ad_group_ad` so I'm just going to remove everything associated with adding image assets.